pkg/cmd/cli/describe/printer.go

```diff
@@ -1131,3 +1139,34 @@ func printRoleBindingRestrictionList(list *authorizationapi.RoleBindingRestricti
 	}
 	return nil
 }
+
+func printPspReview(pspreview *securityapi.PodSecurityPolicyReview, w io.Writer, options kctl.PrintOptions) error {
+	for _, allowedSA := range pspreview.Status.AllowedServiceAccounts {
```

Comment:
btw. name methods, `printPSPReview*`, PSP here is an acronym, so it should be spelled like one.